Source/WebCore/page/scrolling/ScrollAnchoringController.h

```diff
@@ -41,12 +41,18 @@ class ScrollAnchoringController final {
     void invalidateAnchorElement();
     void updateScrollPosition();
     void selectAnchorElement();
+    void chooseAnchorElement();
+    void updateAnchorElement();
+    void printAnchorElement();
     LocalFrameView& frameView() { return m_frameView; }
 
 private:
     LocalFrameView& m_frameView;
+    FloatSize computeOffset(RenderBox& candidate);
     WeakPtr<Element, WeakPtrImplWithEventTargetData> m_anchorElement;
-    FloatPoint m_lastPositionForAnchorElement;
+    FloatSize m_lastOffsetForAnchorElement;
+    bool m_midAdjustingScrollPositionForAnchorElement { false };
```

Comment:
not sure what "midAdjusting" is.

Reply:
Changed to m_midUpdatingScrollPositionForAnchorElement. This indicates that we are currently updating the scroll position in `ScrollAnchoringController::updateScrollPosition` and want to avoid invalidating the anchor element for a scroll caused by a scroll anchoring adjustment.